packages/flutter_tools/test/utils_test.dart

```diff
@@ -79,4 +80,36 @@ baz=qux
       }
     });
   });
+
+  group('Version', () {
+    test('can parse and compare', () {
```

Comment:
`new Version(null, null, null).toString()` might lead to trouble?

If you get more tests, you may want to split your single `test` into atoms.